base/reflection.jl

```diff
@@ -542,14 +546,16 @@ code_llvm(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types)
 code_llvm_raw(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types, false)
 
 """
-    code_native([io], f, types)
+    code_native([io], f, types, [syntax])
```

Comment:
currently if you don't include `io` this will fail, i.e.
```julia
code_native(sin, (Float32,), :intel)
```
vs
```julia
code_native(STDOUT, sin, (Float32,), :intel)
```